libsolidity/ast/AST_accept.h

```diff
@@ -403,7 +403,8 @@ void Block::accept(ASTVisitor& _visitor)
 void Block::accept(ASTConstVisitor& _visitor) const
 {
 	if (_visitor.visit(*this))
-		listAccept(m_statements, _visitor);
+		if (m_statements.size() > 0)
```

Comment:
Why prevent the iteration here?

Reply:
There had been a SEGV as a result of an empty m_statements when we hit an error. However current testing does not show that. So I've removed this. If we hit a SEGV in the future, I will deal with that then.